flow/layers/transform_layer.cc

```diff
@@ -42,12 +41,12 @@ void TransformLayer::Diff(DiffContext* context, const Layer* old_layer) {
 
 void TransformLayer::Preroll(PrerollContext* context) {
   auto mutator = context->state_stack.save();
-  mutator.transform(transform_);
+  mutator.transform(transform_.asM33());
```

Comment:
We should probably file a bug for this, since it'll affect platform views.

Reply:
Specifically a but tracking whether we can make the mutator stack work with full 4x4 matrices.

Reply:
> 

Yes, I agree. I've found the platform views will affect if it comes to a 4x4 transform which is only an identity.